modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -228,14 +300,16 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
     }
 
     // Run the script
+#if defined(USE_GAPI_TBB_EXECUTOR)
+    tbb::concurrent_priority_queue<parallel::tile_node* , parallel::tile_node_indirect_priority_comparator> q {start_tasks.begin(), start_tasks.end()};
+    parallel::execute(q);
+#else
     for (auto &op : m_ops)
     {
-        // (5)
-        Input i{m_res, op.in_objects};
-        Output o{m_res, op.out_objects};
-        op.isl_exec->run(i, o);
+        // (5) and  (6)
+        run_op(op, m_res);
     }
-
+#endif
```

Comment:
Instead of modifying the existing class, can we just put a new one on side? Then there's no changes, just additions to.